src/Microsoft.ML.PCA/PCACatalog.cs

```diff
@@ -35,5 +36,25 @@ public static class PcaCatalog
         /// <param name="columns">Input columns to apply PrincipalComponentAnalysis on.</param>
         public static PrincipalComponentAnalysisEstimator ProjectToPrincipalComponents(this TransformsCatalog.ProjectionTransforms catalog, params PrincipalComponentAnalysisEstimator.ColumnInfo[] columns)
             => new PrincipalComponentAnalysisEstimator(CatalogUtils.GetEnvironment(catalog), columns);
+
+        public static RandomizedPcaTrainer RandomizedPca(this AnomalyDetectionCatalog.AnomalyDetectionTrainers catalog,
```

Comment:
Needs xml docs with remarks and links to a sample. Here or add to #1209 .

Reply:
added xml docs.

sample adding can be part of overall documentation effort #1209

---
In reply to: [255585449](https://github.com/dotnet/machinelearning/pull/2390#discussion_r255585449) [](ancestors = 255585449)